systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -304,4 +306,36 @@ public void testDeployKafkaOnPersistentStorage() {
         assertThat(configMap, valueOfCmEquals("KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", "5"));
         assertThat(configMap, valueOfCmEquals("KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR", "5"));
     }
-}
+
+    @Test
+    @KafkaCluster(name = "my-cluster", kafkaNodes = 1, zkNodes = 2, config = {
+            @CmData(key = "zookeeper-healthcheck-delay", value = "30"),
```

Comment:
We already have a test that checks deployment with custom parameters `testClusterWithCustomParameters()`. Is it necessary to have custom parameters in this case for pre-conditions? WDYT?

Reply:
I think that having custom parameters is necessary because the template can be changed and in this test I'd like to be sure that value was changed from some value to another value.
Maybe I can use just one or two custom parameters.